orchagent/portsorch.cpp

```diff
@@ -3676,6 +3682,15 @@ void PortsOrch::doLagMemberTask(Consumer &consumer)
             continue;
         }
 
+        /* Fast failure if a port type is not a valid type for beeing a LAG member port.
```

Comment:
Could you please revisit the comment? What is fast failure? Also beeing -> typo

Reply:
Fixed